Magical-Kitties/Magical-Kitties-Save-the-Day.html

```diff
@@ -1,173 +1,126 @@
 <div class="container">
+  <button type="roll" class="button-roll" style="transform: translate(15px, 5px);" value="&{template:default} {{name=**Extra Dice**}} {{ Rolled ?{Extra Dice|1} extra dice.}} {{Successes = [[(?{Extra Dice})d6>?{Task Difficulty|Typical,4|Easy,3|Hard,5|Extreme,6}]] }}" data-i18n="extra-dice-rolls-u" data-i18n-title="extra-dice-rolls-u"></button>
+  <img class="kitty-token" name="attr_character_token" data-i18n-title="default-token-click-edit-to-update">
+  <div class="masthead">
+    <span class="magical-kitties">Magical Kitties</span>
+    <span class="save-the-day">Save the Day!</span>
+  </div>
+  <h2 class="whiteonredtext" data-i18n="your-kitty-u"></h2>
+  <div class="redtext width200" data-i18n="kitty-name-u"></div>
+  <input type="text" class="grid-grow-2 input-large" name="attr_character_name">
 
```

Comment:
due to the number of indentation changes, it's much easier to review the diff with [blankspace changes ignored](https://github.com/Roll20/roll20-character-sheets/pull/11500/files?diff=unified&w=1)